### ISSUE TITLE
Conflicting packages and --force-latest

### ISSUE THREAD

**Issue by OliverJAsh** [yatskevich/grunt-bower-task#7]
Bower was recently updated to add the [`--force-latest`](https://github.com/twitter/bower/issues/214) flag to resolve conflicts.

If I run `grunt bower` or just `bower install`, I get:

```
mismatch The version specified in the component.json of package jquery mismatches the tag (1.8.3 vs 1.9.0)
mismatch You should report this problem to the package author
Fatal error: No resolvable dependency for: jquery
```

If I run `bower install --force-latest`, however, I get:

```
Please note that
    Reader requires jquery ~1.9.0
    bootstrap requires jquery ~1.8.0

Resorting to using latest version, jquery v1.9.0.
Conflicts may occur.
```

And then the install continues.

What can be done to incorporate this into `grunt-bower-task`?

**Comment by Iristyle** [comment]
+1, this is a really big deal @yatskevich

**Comment by yatskevich** [comment]
I'll take a look at this in a couple of days. One can add `forceLatest` config parameter to `grunt-bower-task` or better add a capability to pass _any_ Bower parameters via grunt task configuration.

**Comment by Iristyle** [comment]
Yeah, the better route is to just add an options object that passes any Bower install options along.  Should only be a few lines of code really

https://github.com/twitter/bower/blob/master/lib/commands/install.js#L23

**Comment by yatskevich** [comment]
@OliverJAsh, @Iristyle `grunt-bower-task` uses Bower of version 0.8.5 starting from the version 0.1.1. This Bower's version handles conflicting dependencies liberally.

Components declared in the following `component.json` are installed successfully with a warning:

```
Please note that
    grunt-bower-task requires jquery 1.9.1
    bootstrap requires jquery ~1.8.0

Resolved to jquery v1.9.1, which matches the requirement defined in the project's component.json.
Conflicts may occur.
```

**component.json**

``` js
{
  "name": "confliting-deps",
  "version": "0.0.0",
  "dependencies": {
    "jquery": "1.9.1",
    "bootstrap": "2.2.1"
  }
}
```

It seems that `--force-latest` is the default behavior and it is impossible to override using neither `--force-latest false`, nor `--no-force-latest`.
